lib/recurly/bus.js

```diff
@@ -81,37 +67,53 @@ export class Bus extends Emitter {
    * @public
    * @param {String} event name of the message's event
    * @param {Object} body message contents
+   * @param {Object} [options]
+   * @param {Array} [options.exclude] a list of
    */
   send (event, body, options = { exclude: [] }) {
-    const message = new Message(event, body);
-    const {exclude} = options;
+    const { groupId } = this;
+    const message = new Message(event, body, groupId);
+    const { exclude } = options;
 
     this.debug(`sending message to ${this.recipients.length} recipients`, message);
     this.recipients.forEach(r => {
+      if (~exclude.indexOf(r)) return;
```

Comment:
Exclusions were ignored before, which was of no operational consequence, but this reduces noise.